lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -378,16 +379,17 @@ export class PublicClientApplication implements IPublicClientApplication {
 export async function createNestablePublicClientApplication(
     configuration: Configuration
 ): Promise<IPublicClientApplication> {
+    const correlationId = createNewGuid();
```

Comment:
@sameerag Notable in case this changes any of our logging in NAA

Reply:
We create one if not available in token requests in NAA. Ideally this should not break anything. Only concern is the reuse for every token request in since this is a NAA instance. I will check the rest of the PR if we are doing that (hopefully not).

Reply:
This `correlationId` is used for NAA initialization only.